src/utils/tests/frame.spec.js

```diff
@@ -118,4 +119,135 @@ describe("function names", () => {
       });
     });
   });
+
+  describe("collapseFrames", () => {
+    it("collapses React component render frames and Webpack bootstrap frames", () => {
+      const frames = [
```

Comment:
what is the minimum information that we need for this to run? do we need id and display name for example? maybe we can either reduce this so that it only shows relevant info for the test, or we can create a factory that will generate frames for us when needed. something like https://github.com/devtools-html/debugger.html/blob/5026582ce0547f3a964b31cfd53aee1059db3cc6/src/actions/tests/expressions.spec.js#L109 but more general and that could replace other uses of this for tests

Reply:
"library" might be the only one absolutely necessary. I'll look into that.

For the record, I was trying strike a balance between having minimal data and data that resembled what you would actually see in the wild. But for Jest tests I guess it does make sense to only worry about the former.